theseus_gui/src/components/SideNav.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="nav-container">
+    <slot />
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.nav-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 210px;
+  height: 100vh;
+  box-shadow: 0px 0px 8px 3px #00000026;
```

Comment:
Use this shadow instead: ` box-shadow: var(--shadow-inset-sm), var(--shadow-floating);`